config/pitest-suppressions/pitest-imports-suppressions.xml

```diff
@@ -54,14 +54,14 @@
 
 
 
-  <mutation unstable="false">
-    <sourceFile>ImportOrderCheck.java</sourceFile>
-    <mutatedClass>com.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck</mutatedClass>
-    <mutatedMethod>beginTree</mutatedMethod>
-    <mutator>org.pitest.mutationtest.engine.gregor.mutators.experimental.MemberVariableMutator</mutator>
-    <description>Removed assignment to member variable lastImportStatic</description>
-    <lineContent>lastImportStatic = false;</lineContent>
-  </mutation>
+
+
+
+
+
+
+
+
```

Comment:
Why are we adding empty lines to this file?

Reply:
To avoid Conflicts

Reply:
Is the plan to remove them at some point if we can't remove the entire file?

Reply:
Yes, If the entire file is not removed then I will update this and remove it. so at the, it looks clean

But right know in this file 4 mutations are still left which is possible to cover